modules/commands.py

```
@@ -0,0 +1,442 @@
+
+        # Generate shopping list
+        shopping_list = generator.generate_shopping_list_json(
+            args.controller,
+            args.width,
+            args.height,
+            data_pin=args.data_pin,
+            psu=args.psu,
+        )
+
+        shopping_filename = (
+            f"shopping_list_{args.controller}_{args.width}x{height}.json"
```

Ruff (0.12.2)

Undefined name `height`

(F821)